core-bundle/tests/Picker/PickerTest.php

```diff
@@ -98,7 +98,7 @@ public function testReturnsNullAsCurrentUrlIfThereIsNoCurrentMenuItem(): void
         $config = new PickerConfig('page');
         $picker = new Picker($factory, [$provider], $config);
 
-        $this->assertNull($picker->getCurrentUrl());
+        $this->assertSame('', (string) $picker->getCurrentUrl());
     }
```

Comment:
I don’t understand the code behind that test. @aschempp is my fix correct here?

The change from `null` to `''` is because the router cannot return `null` anymore in Symfony 6 I think. https://github.com/symfony/symfony/blob/7eaefd2245ae9f0ac4e47cc42ce0e287c489cc0e/src/Symfony/Component/Routing/Generator/UrlGeneratorInterface.php#L79

Reply:
The change is correct IMHO.